dev/dll/Density/CompactDatePickerTimePickerFlyout.xaml

```diff
@@ -0,0 +1,14 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary 
+  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
+  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"> 
+    <x:Double x:Key="TimePickerFlyoutPresenterItemHeight">32</x:Double>           
+    <x:Double x:Key="TimePickerFlyoutPresenterHighlightHeight">32</x:Double>
```

Comment:
Why is this file separate? What is it for? How do I use it?

Reply:
We have no way to use CompactDatePickerTimePickerFlyout.xaml directly right now.

Reply:
Ok, then what is it for? Why is it not just part of Compact.xaml?

Reply:
I see you have something in the description that this doesn't work. Let's leave the file out of this PR, then. We shouldn't include a resource or file in the project that doesn't work.

Reply:
This parameters are tuned parameter without using winui, and it's useful when we implement final Compact. I will keep this file and add a comment on it.